cmd/kubeadm/app/phases/certs/renewal/manager.go

```diff
@@ -421,50 +400,3 @@ func certToConfig(cert *x509.Certificate) certutil.Config {
 		Usages: cert.ExtKeyUsage,
 	}
 }
-
-func loadCertConfigMutators(certBaseName string) []certConfigMutatorFunc {
-	// TODO: Remove these mutators after the organization migration is complete in a future release
-	// https://github.com/kubernetes/kubeadm/issues/2414
-	switch certBaseName {
-	case kubeadmconstants.EtcdHealthcheckClientCertAndKeyBaseName,
-		kubeadmconstants.APIServerEtcdClientCertAndKeyBaseName:
-		return []certConfigMutatorFunc{
-			removeSystemPrivilegedGroupMutator(),
-		}
-	case kubeadmconstants.APIServerKubeletClientCertAndKeyBaseName:
-		return []certConfigMutatorFunc{
-			removeSystemPrivilegedGroupMutator(),
-			addClusterAdminsGroupMutator(),
-		}
-	}
```

Comment:
let's remove only this code that i selected above and the two functions removeSystemPrivilegedGroupMutator addClusterAdminsGroupMutator.

keep the rest of the mutator code, in the future we might need it.

Reply:
+1

Reply:
all right